scripts/start.mjs

```diff
@@ -0,0 +1,35 @@
+import { spawn, exec } from "child_process";
+console.log("Starting a hardhat node...");
+const hardhat = spawn("yarn contracts hardhat node", { shell: true });
+
+hardhat.stdout.on("data", (data) => {
+    // when start an Account list
+    if (data.toString().indexOf("========") !== -1) {
+        // deploy a Unirep contract and the attester contract
+        exec("yarn contracts deploy", (err) => {
```

Comment:
This can be wrapped in a promise to avoid the callback

```js
await new Promise((rs, rj) => exec('yarn contracts deploy', (err) => err ? rj(err) : rs()))
```

Reply:
If there is an error you can catch and `process.exit(1)`